zproject/default_settings.py

```diff
@@ -437,3 +437,6 @@ class JwtAuthKey(TypedDict):
 
 # Use half of the available CPUs for data import purposes.
 DEFAULT_DATA_EXPORT_IMPORT_PARALLELISM = (len(os.sched_getaffinity(0)) // 2) or 1
+
+# GIPHY API key
+GIPHY_API_KEY = get_secret("giphy_api_key")
```

Comment:
Let's put this next to similar settings -- I think in this case, those for other integrations like Twitter/Jitsi/etc.

It also belongs in prod_settings_template.py in the same section as Twitter keys.

Reply:
Okay